app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt

```diff
@@ -179,16 +179,19 @@ class StoryActivityTest {
   class TestApplication : Application(), ActivityComponentFactory, ApplicationInjectorProvider {
     private val component: TestApplicationComponent by lazy {
       DaggerStoryActivityTest_TestApplicationComponent.builder()
-        .setApplication(this)
+        .setApplication(context = this)
```

Comment:
this is incorrect. 
PTAL - https://kotlinlang.org/docs/functions.html#named-arguments

Reply:
> @jkugsiya Look at line number 116, these are the places where we need named arguments most, as passing simply 1 or any number doesn't help us understand what is the requirement of that argument.

As you said in line number 116 we need named arguments the most, but Android Studio is not allowing to make it a named argument by throwing an error that named arguments are not allowed in non-kotlin functions.
![Screenshot 2021-03-16 135535](https://user-images.githubusercontent.com/42642325/111278223-502be080-865f-11eb-94d8-4a9e2ce6f8e8.jpg)

Reply:
ah, what I missed here is `actionOnItemAtPosition` is a java function. Thanks for the update.